File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/cadastro/perfil/PerfilModuloDTO.java
package br.com.grupopipa.gestaointegrada.cadastro.perfil;

import java.util.UUID;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PerfilModuloDTO {

    private UUID id;
    private UUID perfilId;
    private UUID moduloId;
    private String moduloNome;
    private boolean podeListar;
    private boolean podeVisualizar;
    private boolean podeEditar;
    private boolean podeDeletar;

}

File: src/frontend/src/app/components/cadastro/perfil/model/perfil-modulo-dto.ts
export interface PerfilModuloDTO {
  id?: string;
  perfilId: string;
  moduloId: string;
  moduloNome: string;
  podeListar: boolean;
  podeVisualizar: boolean;
  podeEditar: boolean;
  podeDeletar: boolean;
}
